src/ranges.hh

```diff
@@ -574,6 +575,13 @@ bool contains(Range&& range, const T& value)
     return find(range, value) != end(range);
 }
 
+template<typename T, typename U>
+bool contains(std::initializer_list<T> range, const U& value)
```

Comment:
I would rather avoid having to introduce a copy of each algorithm for initializer_list, so until we find a solution that does not requires duplication I'd rather explicitely pass Array{x, y}

Reply:
Makes sense, I went with the safe choice of `... == ... or ... == ...` now, which we use in lots of places.